src/aosm/azext_aosm/_configuration.py

```diff
@@ -13,10 +20,14 @@
         "Name of the Publisher resource you want your definition "
         "published to. Will be created if it does not exist."
     ),
+    "publisher_name_nsd": (
+        "Name of the Publisher resource you want your definition published to."
+    ),
+    "publisher_resource_group_name_nsd": ("Resource group for the Publisher resource."),
     "nf_name": "Name of NF definition",
     "version": "Version of the NF definition",
     "acr_artifact_store_name": "Name of the ACR Artifact Store resource. Will be created if it does not exist.",
-    "location": "Azure location to use when creating resources",
+    "location": "Azure location to use when creating resources.",
     "blob_artifact_store_name": "Name of the storage account Artifact Store resource. Will be created if it does not exist.",
     "artifact_name": "Name of the artifact",
     "file_path": (
```

Comment:
as discussed in person, we need to decide whether to keep this and add to nfds or remove it and just create in the user's current directory

Reply:
For cnfs, there is no arm template provided and it just makes it in the current directory. I think for consistency we should just make it in the current directory